package.json

```diff
@@ -66,8 +66,11 @@
     "xdr": "0.5.3"
   },
   "devDependencies": {
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
This is needed for babel-jest in addition to the version in dependencies, see https://jestjs.io/docs/en/getting-started.html#using-babel